presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java

```diff
@@ -814,6 +814,9 @@ public void testRound()
         assertFunction("round(REAL '90000000.0', 1000)", REAL, 90000000.0f);
         assertFunction("round(REAL '300000000.0', 100)", REAL, 300000000.0f);
         assertFunction("round(REAL '-300000000.0', 100)", REAL, -300000000.0f);
+        assertFunction("round(REAL '1.0E19', 1)", REAL, 1.0E19f);
```

Comment:
These should probably go in a new test method with a comment linking to the bug so they can pass or fail independently